packages/data-broker/src/services/dhis/DhisInputSchemeResolvingApiProxy.js

```diff
@@ -97,7 +97,7 @@ export class DhisInputSchemeResolvingApiProxy {
 
     for (const dataElementCode of dataElementCodes) {
       const dataElement = dataElements.find(d => d.code === dataElementCode);
-      if (!dataElement.config.dhisId) {
+      if (!dataElement?.config?.dhisId) {
```

Comment:
I think you've got the wrong base branch @nick-beyondessential?

Reply:
yes, just changed